presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -95,3 +95,13 @@ Array Functions
 
     Subsets array ``x`` starting from index ``start`` (or starting from the end
     if ``start`` is negative) with a length of ``length``.
+
+.. function:: zip(array1, array2[, array3[, array4]]) -> array<row>
+
+    | Aggregate elements of the arrays passed as arguments into rows. Similar to the combined use of ``UNNEST`` and ``ARRAY_AGG``.
```

Comment:
I'm not sure people will understand the reference to ``UNNEST`` as that syntax is a bit esoteric. I would word this more like "Merges the given arrays, element-wise, into a single array of tuples"